data_base/squema.sql

```diff
@@ -38,4 +38,23 @@ CREATE TABLE album (
   item_id  INT REFERENCES item(id),
   on_spotify  BOOLEAN,
   PRIMARY KEY(item_id)
+);
+
+CREATE TABLE label(
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  title VARCHAR(32),
+  color VARCHAR(20),
+);
```

Comment:
Please, reorder the creation of tables. The table label is referenced before to was created